src/modules/wizeng/WizengModule.v3

```diff
@@ -53,6 +53,25 @@ class WizengModule extends HostModule("wizeng") {
 				var imported_sig = Canon.sigPR(SigCache.arr_ii, [rtype]);
 				result[j] = HostFunction.new("new_funcref", imported_sig, func);
 			}
+			if (!found && Strings.equal("puts_array", i.imp.field_name)) {
+				var params = impf.sig.params;
+				if (params.length != 3) continue;
```

Comment:
We also need to check the types of params 1 and 2.

Reply:
Since I did not use `params` as the parameter list of the function signature, I thought I did not need to. Fixed.